packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -959,8 +959,8 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
     }
   }
 
-  /// This method can be called to filter the list of available semantic actions,
```

Comment:
The changes in this file should not be reverted, I think. They are fixing another unrelated problem, no?